src/components/reactTasks/dislike.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+export default class Disike extends React.Component {
+  constructor() {
+    super()
+    this.state = {disliked: true}
+    this.handleClick = this.handleClick.bind(this)
```

Comment:
This is the way they do it in the React docs, but we usually do it as you've shown in the file above (`src/components/reactTasks/comment.js`) where instead of having `...bind(this)` in the constructor, you do what you've done on line 28 ^^.

Reply:
again it's good to be consistent. All should be done one way or the other. But I would go with the way Alex said